src/plugins/unified_search/public/filters_builder/filter_item/filter_item.styles.ts

```diff
@@ -26,9 +26,13 @@ export const fieldAndParamCss = (euiTheme: EuiThemeComputed) => css`
   .euiFormRow {
     max-width: 800px;
   }
-  &:focus-within {
+  // Expand the combobox if the dropdown is open or the input has focus
+  &:has(input[aria-expanded='true'], input:focus) {
```

Comment:
Quick note: I'm aware that latest production Firefox _currently_ does not support `:has` ([caniuse link](https://caniuse.com/css-has)), however it will by release 120 (slated for mid December).

IMO, this is reason enough to move forward with `:has()` especially as I see the flex width UX as a progressive enhancement (it's nice to have to show more text, but not a must have).